device-types/Nokia/7210-SAS-M-24F-2XFP.yaml

```diff
@@ -0,0 +1,75 @@
+---
+manufacturer: Nokia
+model: 7210 SAS-M 24F 2XFP
+slug: nokia-7210-sas-m-24f-2xfp
+part_number: 3HE05828AAAB01
+u_height: 2
+is_full_depth: false
+airflow: side-to-rear
+weight: 5
+weight_unit: kg
+console-ports:
+  - name: Console
+    type: rj-45
+interfaces:
+  - name: 1/1/1
+    type: 1000base-x-sfp
+  - name: 1/1/2
+    type: 1000base-x-sfp
+  - name: 1/1/3
+    type: 1000base-x-sfp
+  - name: 1/1/4
+    type: 1000base-x-sfp
+  - name: 1/1/5
+    type: 1000base-x-sfp
+  - name: 1/1/6
+    type: 1000base-x-sfp
+  - name: 1/1/7
+    type: 1000base-x-sfp
+  - name: 1/1/8
+    type: 1000base-x-sfp
+  - name: 1/1/9
+    type: 1000base-x-sfp
+  - name: 1/1/10
+    type: 1000base-x-sfp
+  - name: 1/1/11
+    type: 1000base-x-sfp
+  - name: 1/1/12
+    type: 1000base-x-sfp
+  - name: 1/1/13
+    type: 1000base-x-sfp
+  - name: 1/1/14
+    type: 1000base-x-sfp
+  - name: 1/1/15
+    type: 1000base-x-sfp
+  - name: 1/1/16
+    type: 1000base-x-sfp
+  - name: 1/1/17
+    type: 1000base-x-sfp
+  - name: 1/1/18
+    type: 1000base-x-sfp
+  - name: 1/1/19
+    type: 1000base-x-sfp
+  - name: 1/1/20
+    type: 1000base-x-sfp
+  - name: 1/1/21
+    type: 1000base-x-sfp
+  - name: 1/1/22
+    type: 1000base-x-sfp
+  - name: 1/1/23
+    type: 1000base-x-sfp
+  - name: 1/1/24
+    type: 1000base-x-sfp
+  - name: 1/1/25
+    type: 10gbase-x-xfp
+  - name: 1/1/26
+    type: 10gbase-x-xfp
+  - name: 1/2/1
+    type: 10gbase-x-xfp
+  - name: 1/2/2
+    type: 10gbase-x-xfp
+module-bays:
+  - name: mda2
+  - name: Power Supply 1
+  - name: Power Supply 2
+  - name: Fan Tray 1
```

Comment:
Generally you should also be adding `position` attributes to module-bays. When a module is imported in NetBox, you will need to use the position value

This is done automatically when importing a module that has its interface/port name that contains `{module}`.